lib/Server.js

```diff
@@ -130,12 +130,13 @@ class Server {
         next(err);
       });
 
-      return new Promise(resolve => {
+      const listenPromise = new Promise(resolve => {
         this.httpServer = this.server.listen(this.port, () => {
           console.log(`Server listening at http://${this.hostname}:${this.port}`);
           resolve();
         });
       });
+      return Promise.all([listenPromise, readyPromise]);
```

Comment:
`Server.start  is called by `program` (`commander`) in `/bin/rnws.js`. Based on https://github.com/tj/commander.js/issues/352 returning a `Promise` here isn't doing anything anyway, so I don't see a lot of value in this change.